zebra/zebra_script.c

```diff
@@ -428,6 +428,8 @@ void lua_pushzebra_dplane_ctx(lua_State *L, const struct zebra_dplane_ctx *ctx)
 	case DPLANE_OP_NONE:
 	case DPLANE_OP_STARTUP_STAGE:
 	case DPLANE_OP_VLAN_INSTALL:
+
```

Comment:
There are more than a few places that have just whitespace changes.  Please go through your whole commit  remove them.